src/libs/OptionsListUtils.ts

```diff
@@ -492,7 +496,7 @@ function getIOUReportIDOfLastAction(report: OnyxEntry<Report>): string | undefin
  * Get the last message text from the report directly or from other sources for special cases.
  */
 function getLastMessageTextForReport(report: OnyxEntry<Report>, lastActorDetails: Partial<PersonalDetails> | null, policy?: OnyxEntry<Policy>): string {
-    const reportID = report?.reportID ?? '-1';
+    const reportID = report?.reportID ?? String(CONST.DEFAULT_NUMBER_ID);
```

Comment:
Looking at [this](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids) we should never default to string value cc @VickyStash 
```suggestion
    const reportID = report?.reportID;
```

Reply:
@mountiny Correct! 
Also, this file is in the [list of ignored files](https://github.com/Expensify/App/blob/a6ea6a1f3a8ace0bc0a6b6c26e666627b697ea22/.eslintrc.changed.js#L18), so you should be able to skip id updates here and the lint will pass on its own.

Reply:
@JKobrynski would you be able to update this please?